pyvrp/cpp/Solution_bindings.cpp

```diff
@@ -70,6 +72,14 @@ PYBIND11_MODULE(_Solution, m)
         });
 
     py::class_<Solution>(m, "Solution")
+        // Note, order of constructors is important, since Route implements
+        // __len__ and __index__, it can also be converted to std::vector<int>
```

Comment:
How does `Route` implement `__index__`?

Reply:
`Client Solution::Route::operator[](size_t idx) const { return visits_[idx]; }`. I'll clarify by calling it `Solution::Route` (I think that's your issue?).

Reply:
No, my issue is that that dunder is called `__getitem__`, not `__index__`. The index dunder is fairly specific, and not something we implement AFAICT (I don't think we get it by default, and we certainly don't specify it ourselves).